pychatml/alpaca/alpaca.py

```diff
@@ -1,6 +1,11 @@
 from typing import List, Dict
+from deprecated import deprecated
 
 
+@deprecated(
```

Comment:
why deprecate those and not just have them init the converter?

Reply:
In case people have already written code using the current structure.